site/_apps/nuxt-hn.txt

```diff
@@ -0,0 +1,29 @@
+---
+weight: 15
+title:  'HNPWA with Nuxt.js'
+github-title: 'nuxt/hackernews'
+libraries: 
+    - name: 'Vue'
+    - name: 'Nuxt'
+    - name: 'PWA module'
+module-bundling: 'Webpack'
+service-worker: 'Workbox'
+perfomance-patterns: 'PRPL, Prefetch/Preload JS + DNS + Data, Critical path CSS, Server side caching'
+server-side-rendering: 'Yes'
+api: 'Hacker News Firebase API'
+hosting: 'Now + CloudFlare CDN'
+authors: 
+    - name: 'Pooya'
+    - name: 'Sebastien'
+    - name: 'Alexandre'
+lighthouse: '91/100'
+interactive-em: '2.8s'
+interactive-faster-3g: '2.4s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170620_PG_a2a9feaf4ace07a61b2c6c2a171b1c79&run=1
+wpt-em-link: https://www.webpagetest.org/result/170620_AM_f5e1ae2f305c1d81b44cfe0c9c930e35
```

Comment:
Looks like the interactive times are do not quite match the report, and there was an issue with the main thread settling in the EM_3G test.

Started some new tests here:
em - https://www.webpagetest.org/result/170620_2Y_5f5fdbaae7d479adf52986a98fc8a720/
faster - https://www.webpagetest.org/result/170620_WP_e35e74fdc07b32752e1b642cf8e159f9/

Will take a look after they complete.